kDrive/UI/Controller/Files/Search/SearchFiltersViewController.swift

```diff
@@ -99,6 +107,41 @@ class SearchFiltersViewController: UITableViewController {
         }
     }
 
+    // MARK: - TextFieldDelegate
```

Comment:
Maybe we can move the delegate conformance in an extension?

Reply:
sure.